jest-puppeteer.config.js

```diff
@@ -1,10 +1,10 @@
 module.exports = {
   launch: {
     args: ['--no-sandbox', '--disable-setuid-sandbox'],
-    executablePath: process.env.PUPPETEER_EXECUTABLE_PATH,
+    headless: 'new',
```

Comment:
Using the new / upcoming headless Chrome mode, which uses Chrome for Testing instead of Chromium.

<img width="601" alt="image" src="https://github.com/commercetools/merchant-center-application-kit/assets/1110551/31ed3420-fe8a-4ca2-94b1-d6cdb9719a96">